catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/CswRecordMetacardType.java

```diff
@@ -432,14 +430,17 @@ private void addCswSummaryRecordSpecificAttributes() {
 
         addDateSubstitutionNames();
 
-        descriptors
-                .add(new AttributeDescriptorImpl(CSW_DESCRIPTION, NON_QUERYABLE /* indexed */, true /* stored */,
-                        false /* tokenized */, true /* multivalued */, BasicTypes.STRING_TYPE));
+        descriptors.add(new AttributeDescriptorImpl(CSW_DESCRIPTION, NON_QUERYABLE /* indexed */, true /* stored */,
```

Comment:
I was wondering about this as well.  After a little investigation, the "description" field is queryable (true) in the Basic Metacard Type.  It should follow that all of the "description" mappings in the CswRecordMetacardType should also be queryable.  Changed.